src/citrine/resources/data_concepts.py

```diff
@@ -388,6 +388,102 @@ def register_all(self, models: List[ResourceType], dry_run=False) -> List[Resour
         )
         return [self.build(obj) for obj in response_data['objects']]
 
+    def async_update(self, model: ResourceType,
+                     dry_run: Optional[bool] = False,
+                     wait_for_response: Optional[bool] = True,
```

Comment:
```suggestion
                     wait_for_response: bool = True,
```
This should never actually be None.